tf/modules/ooni_dev_user/main.tf

```diff
@@ -0,0 +1,188 @@
+        "ecr-public:ListTagsForResource",
+        "ecr:BatchCheck*",
+        "ecr:BatchGet*",
+        "ecr:Describe*",
+        "ecr:Get*",
+        "ecr:List*",
+        "ecs:Describe*",
+        "ecs:List*",
+        "elasticloadbalancing:Describe*",
+        "logs:Describe*",
+        "logs:FilterLogEvents",
+        "logs:Get*",
+        "logs:ListAnomalies",
+        "logs:ListLogAnomalyDetectors",
+        "logs:ListLogDeliveries",
+        "logs:ListTagsForResource",
+        "logs:ListTagsLogGroup",
+        "logs:StartLiveTail",
+        "logs:StartQuery",
+        "logs:StopLiveTail",
+        "logs:StopQuery",
+        "logs:TestMetricFilter",
+        "iam:Generate*",
+        "iam:Get*",
+        "iam:List*",
+        "iam:Simulate*",
+        "rds:Describe*",
+        "rds:Download*",
+        "rds:List*",
+        "route53-recovery-cluster:Get*",
+        "route53-recovery-cluster:ListRoutingControls",
+        "route53-recovery-control-config:Describe*",
+        "route53-recovery-control-config:GetResourcePolicy",
+        "route53-recovery-control-config:List*",
+        "route53-recovery-readiness:Get*",
+        "route53-recovery-readiness:List*",
+        "route53:Get*",
+        "route53:List*",
+        "route53:Test*",
+        "route53domains:Check*",
+        "route53domains:Get*",
+        "route53domains:List*",
+        "route53domains:View*",
+        "route53resolver:Get*",
+        "route53resolver:List*",
+        "s3:DescribeJob",
+        "s3:Get*",
+        "s3:List*",
+        "secretsmanager:Describe*",
+        "secretsmanager:GetResourcePolicy",
+        "secretsmanager:List*",
+        "servicediscovery:DiscoverInstances",
+        "servicediscovery:DiscoverInstancesRevision",
+        "servicediscovery:Get*",
+        "servicediscovery:List*",
+        "ssm:Describe*",
+        "ssm:Get*",
+        "ssm:List*",
+        "states:Describe*",
+        "states:GetExecutionHistory",
+        "states:List*",
+        "vpc-lattice:GetAccessLogSubscription",
+        "vpc-lattice:GetAuthPolicy",
+        "vpc-lattice:GetListener",
+        "vpc-lattice:GetResourcePolicy",
+        "vpc-lattice:GetRule",
+        "vpc-lattice:GetService",
+        "vpc-lattice:GetServiceNetwork",
+        "vpc-lattice:GetServiceNetworkServiceAssociation",
+        "vpc-lattice:GetServiceNetworkVpcAssociation",
+        "vpc-lattice:GetTargetGroup",
+        "vpc-lattice:ListAccessLogSubscriptions",
+        "vpc-lattice:ListListeners",
+        "vpc-lattice:ListRules",
+        "vpc-lattice:ListServiceNetworks",
+        "vpc-lattice:ListServiceNetworkServiceAssociations",
+        "vpc-lattice:ListServiceNetworkVpcAssociations",
+        "vpc-lattice:ListServices",
+        "vpc-lattice:ListTagsForResource",
+        "vpc-lattice:ListTargetGroups",
+        "vpc-lattice:ListTargets"
+    ]
+    resources = ["*"]
+  }
+}
+
+resource "aws_iam_user_policy" "ooni_tf_dev" {
+  name = "OONITFDevPolicy"
```

Comment:
I would suggest not putting `dev` in the name, since we are going to have this role in every environment and it's better to add further specificity by saying that it's about granting access to github.

A better name would be `OONIDevopsGithubPolicy`